index.js

```diff
@@ -16,8 +16,9 @@ function create(options) {
 
     return function redirectSSL(req, res, next) {
         const _isHttps = isHTTPS(req, xForwardedProto)
-
-        if (_isHttps === false || (redirectUnknown && _isHttps === null)) {
+        const isProduction = process.env.NODE_ENV === 'production'
```

Comment:
Would you please add another condition to force redirects using `isProduction = options.redirect === true || ...` ?

Reply:
Or another idea: Moving this check to `defaults` above. 
```js
defaults = {
  redirect: process.env.NODE_ENV === 'production
}
```

And then using that check for shouldRedirect